src/rest-client.ts

```diff
@@ -1151,20 +1050,12 @@ export class RestClient extends BaseRestClient {
     return this.postPrivate('/api/v5/tradingBot/grid/order-algo', params);
   }
 
-  /**
-   * @deprecated Breaking change: this function will change soon to accept a single object for all REST API parameters.
-   * See issue https://github.com/tiagosiebler/okx-api/issues/88 for more details.
-   */
-  amendGridAlgoOrder(
-    algoId: string,
-    instId: string,
-    triggerPx: { slTriggerPx?: numberInString; tpTriggerPx?: numberInString },
-  ): Promise<any[]> {
-    return this.postPrivate('/api/v5/tradingBot/grid/amend-order-algo', {
-      algoId,
-      instId,
-      ...triggerPx,
-    });
+  amendGridAlgoOrder(params: {
+    algoId: string;
+    instId: string;
+    triggerPx: { slTriggerPx?: numberInString; tpTriggerPx?: numberInString };
```

Comment:
Same here with triggerPx, you can see the old request used a spread merge when passing params `...triggerPx`